akka-cluster-sharding/src/multi-jvm/scala/akka/cluster/sharding/ClusterShardingLeavingSpec.scala

```diff
@@ -81,7 +78,7 @@ abstract class ClusterShardingLeavingSpecConfig(val mode: String) extends MultiN
       dir = target/ClusterShardingLeavingSpec/sharding-ddata
       map-size = 10 MiB
     }
-    """))
+    """).withFallback(MultiNodeClusterSpec.clusterConfig))
```

Comment:
didn't I add the properties in `MultiNodeSpec`?
or is it `run-coordinated-shutdown-when-down = off` that is required here?

Reply:
That was the property I was looking at but given these tests do an explicit leave the properties youa dded to `MultiNodeSpec` should be sufficient. Nearly all of our cluster tests do use that as a fallback but not the cluster sharding ones.